Packs/Core/TestPlaybooks/playbook-Local_Analysis_alert_Investigation_-_Test.yml

```diff
@@ -0,0 +1,934 @@
+    scriptarguments:
+      all:
+        simple: "yes"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": -50
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "14":
+    id: "14"
+    taskid: 20a213a1-ff02-4833-8c25-52491f49d409
+    type: title
+    task:
+      id: 20a213a1-ff02-4833-8c25-52491f49d409
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 2050
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "16":
+    id: "16"
+    taskid: a3c6ecfa-c4ae-4401-8dcc-887824844e28
+    type: regular
+    task:
+      id: a3c6ecfa-c4ae-4401-8dcc-887824844e28
+      version: -1
+      name: Error Occured
+      description: Prints an error entry with a given message
+      scriptName: PrintErrorEntry
+      type: regular
+      iscommand: false
+      brand: ""
+    scriptarguments:
+      message:
+        simple: Test Failed
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -60,
+          "y": 1860
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "18":
+    id: "18"
+    taskid: fad71e65-49ee-4608-83f3-ab852a735942
+    type: condition
+    task:
+      id: fad71e65-49ee-4608-83f3-ab852a735942
+      version: -1
+      name: Test core-retrieve file from endpoint
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "16"
+      "yes":
+      - "21"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: containsGeneral
+          left:
+            value:
+              complex:
+                root: ExtractedFiles
+            iscontext: true
+          right:
+            value:
+              simple: calc.exe
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1190
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "19":
+    id: "19"
+    taskid: e653b5a3-f31c-4bfa-848c-f53fdde3faf7
+    type: playbook
+    task:
+      id: e653b5a3-f31c-4bfa-848c-f53fdde3faf7
+      version: -1
+      name: Local Analysis alert Investigation
```

Comment:
Replace it with the same playbook for updating the changes with the input Query.